### ISSUE TITLE
Add release-please

### ISSUE THREAD

**Issue by daine** [cds-snc/design-gc-conception#569]
As a developer, I want to automate release change updates so it takes less time to maintain the product

Repositories that would be nice to have this feature:
- [x] https://github.com/cds-snc/gcds-tokens
  - [x] Secrets added
  - [x] Action added
  - [x] Action tested
- [x] https://github.com/cds-snc/gcds-components
  - [x] Secrets added
  - [x] Action added
  - [x] Action tested
- ~[ ] https://github.com/cds-snc/gcds-docs~
  - ~[x] Secrets added~
  - ~[ ] Action added~
  - ~[ ] Action tested~
- [x] https://github.com/cds-snc/gcds-utility
  - [x] Secrets added
  - [x] Action added
  - [x] Action tested
  

TBD:
- https://github.com/cds-snc/gcds-utility
- https://github.com/cds-snc/gcds-terraform

## Notes
Release please:
https://github.com/googleapis/release-please

## Technical Debt Triage Matrix

**Risk Value Scoring:**
Level | Value
-- | --
High | 3
Medium | 2
Low | 1

**Risk of not implementing automated release documentation and changelog management:**
Technical Debt - Risk Types | Level | Value | Assessment notes
-- | -- | -- | --
System Fault Risk - Risk of system errors or application downtime | Low  |  1 | Risk would be mistakes with changing package versions manually 
Business Area Risk - Risk of business area visibility / damage to user experience | Medium |  2 | Risk would be having outdated release documentation
Developer Fault Risk - How likely will this tech debt cause a future error related to coding on top of it | Medium  |  2 | Risk would be developer (human) mistakes that could arise from the manual update process
Time Scale Risk - Compound risk effect if left alone. How much more difficult to fix or dangerous will this become over time? | Low |  1 | Risk is associated to the velocity of the developers to produce new features and patches each sprint, and documenting the release
Time Sink Risk - How much will this tech debt slow the development process down | Medium/High |  2.5 | Risk is opposite: how will the solution improve and accelerate the development process
**TOTAL SCORE:** |   |  **8.5** |

---
_Epic:_
-  _https://github.com/cds-snc/design-gc-conception/issues/628_

**Comment by daine** [comment]
Setup on `gcds-tokens` has been successful:
https://github.com/cds-snc/gcds-tokens/releases
https://github.com/cds-snc/gcds-tokens/blob/main/CHANGELOG.md

May require some [minor tweaking of the configuration](https://github.com/cds-snc/gcds-tokens/pull/200) to match the devs' previous way of documenting releases

**Comment by daine** [comment]
For `gcds-components`:
https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#using-with-github-actions-and-lerna

**Comment by daine** [comment]
Blocked by https://github.com/cds-snc/design-gc-conception/issues/667

**Comment by daine** [comment]
Skipping release please for `gcds-docs` since we use amplify ci/cd deployments which can redeploy commits, and there is no need for a changelog for that repository.
